### ISSUE TITLE
String in Buttons not working

### ISSUE THREAD

**Issue by lgremme** [formr/Formr#138]
The variable $string in the button is not working.
```
$form = new Formr\Formr('bootstrap');
$form->open('','',$_SERVER['REQUEST_URI']);
$form->csrf();
$form->input_submit('send', '&nbsp;', 'Name', 'send', 'class="btn btn-danger"');
```
The output of the button is the following:
```
<input type="submit" name="send" value="Name" id="send" class="btn btn-primary btn-custom">
```

After some Debug tasks the string isn't set.
The string is overwritten in the file: class.formr.php, line 761, function _fix_classes

```
if($this->_wrapper_is('bootstrap')) {
     $classes = $this->controls['button-primary'];
}
```
I think there will be missed some if cases.
Formr Version: 1.4.2
Thank you for your great work!

**Comment by JohnDBowne** [comment]
I'm seeing the same thing, I think. I just updated from 1.2.4 to 1.4.2. 
```
			$showDataButtonForm = new Formr\Formr('bootstrap3');
			echo $showDataButtonForm->form_open();
			echo $showDataButtonForm->input_submit('modeShowData','','Show Data','',$disableStr .' class="btn btn-primary btn-block"');
			echo $showDataButtonForm->form_close();
```
What I'm seeing in Chrome is: 
`<input type="submit" name="modeShowData" value="Show Data" id="modeShowData" class="btn btn-primary" disabled="">`

Thanks,
    johnbo

**Comment by timgavin** [comment]
This will be fixed in 1.4.3. Thanks!
